web-test-runner.config.mjs

```diff
@@ -0,0 +1,9 @@
+import {esbuildPlugin} from '@web/dev-server-esbuild';
+
+export default {
```

Comment:
we might want to look into other options - running in specific browsers etc, this is the default setting

Reply:
Makes sense! In the future, I see the opportunity here to extract some common config file and import it both upon prod build and here in WTR, so that we avoid the duplicity and risk of diverging. For now I believe it's good enough though. :)